pkg/monitor/worker.go

```diff
@@ -27,6 +28,9 @@ import (
 // nsgMonitoringFrequency is used for initializing NSG monitoring ticker
 var nsgMonitoringFrequency = 10 * time.Minute
 
+// changefeedBatchSize is how many items in the changefeed to fetch in each page
+const changefeedBatchSize = 50
```

Comment:
Would be beneficial to have this as an environment variable so it's configurable without a code redeployment.

Reply:
We do have to retag/rebuild/redeploy VMSSes from scratch at the moment, so I don't know if it would give us a great deal of changeability. But, I'll see if I can put something in.